drivers/usb/usbccgp/misc.c

```diff
@@ -89,11 +32,9 @@ USBCCGP_SyncUrbRequest(
         return STATUS_INSUFFICIENT_RESOURCES;
     }
 
-    /* Initialize event */
-    KeInitializeEvent(&Event, NotificationEvent, FALSE);
-
-    /* Get next stack location */
-    IoStack = IoGetNextIrpStackLocation(Irp);
+    /* Get the first stack location in a newly created IRP */
+    IoSetNextIrpStackLocation(Irp);
```

Comment:
I think you need an extra stack location for this?
TBH, I would probably consider it more obvious to just leave the manual `IoCallDriver`.